benchmark/Streamly/Benchmark/Prelude/Serial.hs

```diff
@@ -991,14 +1111,23 @@ dropAll value n = composeN n $ S.drop (value + 1)
 dropWhileTrue :: MonadIO m => Int -> Int -> SerialT m Int -> m ()
 dropWhileTrue value n = composeN n $ S.dropWhile (<= (value + 1))
 
-{-# INLINE _dropWhileMTrue #-}
-_dropWhileMTrue :: MonadIO m => Int -> Int -> SerialT m Int -> m ()
-_dropWhileMTrue value n = composeN n $ S.dropWhileM (return . (<= (value + 1)))
+{-# INLINE dropWhileMTrue #-}
+dropWhileMTrue :: MonadIO m => Int -> Int -> SerialT m Int -> m ()
+dropWhileMTrue value n = composeN n $ S.dropWhileM (return . (<= (value + 1)))
 
 {-# INLINE dropWhileFalse #-}
 dropWhileFalse :: MonadIO m => Int -> Int -> SerialT m Int -> m ()
 dropWhileFalse value n = composeN n $ S.dropWhile (> (value + 1))
 
+-- XXX Decide on the time interval
```

Comment:
Why can't we decide on that?

Reply:
I had something in mind when I wrote this comment. I'll leave it as it is for now.